pkg/reconciler/pipelinerun/resources/validate_dependencies.go

```diff
@@ -0,0 +1,99 @@
+package resources
```

Comment:
sorry @sbwsg for early review, missing copyright license here and in the test file 😉

Reply:
Ah, great point, I completely forgot, thanks @pritidesai